### PR TITLE
Update use-group-policy.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1834

@kaarins could you kindly confirm this information? Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -553,8 +553,8 @@ This setting lets you specify SharePoint team site libraries to sync automatical
 This feature is not enabled for on-premises SharePoint sites.
 
 > [!IMPORTANT]
-> Do not enable this setting for libraries with more than 5,000 files or folders.
-> Do not enable this setting for the same library to more than 1,000 devices.
+> For optimum performance, we recommend storing no more than 300,000 files across all synced document libraries, even if you're using Files On-Demand or choosing only some folders within the libraries to sync.
+> Do not enable this setting for the same library to more than 1,000 devices, this is a limitation of Windows Push Notification Services (WNS).
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md:557]:
```suggestion
> Do not enable this setting for the same library to more than 1,000 devices. This is a limitation of Windows Push Notification Services (WNS).
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kaarins** [comment]
Our recommendation is to not use this policy on libraries with more than 5,000 items. The guidance should remain as is for now. Thanks!

**Pull request closed without merging**